utils/cliutils/commandsflags.go

```diff
@@ -1375,6 +1377,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  IgnoreState,
 		Usage: "[Default: false] Set to true to ignore the saved state from previous transfer-files operations.` `",
 	},
+	ProxyKey: cli.StringFlag{
+		Name:  ProxyKey,
+		Usage: "[Optional] A proxy key configured in the source Artifactory server to use during the transfer from the source to the target server. Configure this proxy at \"Proxies | Configuration | Proxy Configuration\" in the Administration UI.` `",
```

Comment:
[Optional] A proxy key configured in the source Artifactory server to use during the transfer from the source to the target server. Configure this proxy at \"Proxies | Configuration | Proxy Configuration\" in the Administration UI.
-->
[Optional] The key of an HTTP proxy configuration in Artifactory. This proxy will be used for the transfer traffic between the source and target instances. To configure this proxy, go to \"Proxies | Configuration | Proxy Configuration\" in the JFrog Administration UI.